test/adapter/json_api/links_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+LinkPost = Class.new(Model)
+class LinkPostSerializer < ActiveModel::Serializer
+  belongs_to :link_author
+end
+
+LinkAuthor = Class.new(Model)
+class LinkAuthorSerializer < ActiveModel::Serializer
+  links do
+    link :self, href: "//link_author/#{object.id}"
```

Comment:
@beauby a protocol relative URI still contains the domain. This discussion's url: https://github.com/rails-api/active_model_serializers/pull/1246#discussion_r41348842

Its 'protocol' relative version: //github.com/rails-api/active_model_serializers/pull/1246#discussion_r41348842

Not //rails-api/active_model_serializers/pull/1246#discussion_r41348842